### ISSUE TITLE
The route helper method throws an error when running cbPlaywright on Adobe ColdFusion

### ISSUE THREAD

**Issue by josephmchastain** [coldbox-modules/cbPlaywright#7]
The _route_ helper method in _cbPlaywright.models.PlaywrightMixins_ throws a _"Variable STRUCTVALUEARRAY is undefined."_ error when running cbPlaywright on ACF. As can be deduced from the error message, _structValueArray_ is not a defined function in ACF. (Brad Wood has submitted a [ticket ](https://tracker.adobe.com/#/view/CF-4215606)with Adobe to rectify this.)

The line of code at issue is:
`var pathArray = structValueArray( arguments );` 

The below looks like it works as a replacement until ACF supports the `structValueArray` function (though it feels wrong to replace one line of code with four).

```
var pathArray = [];
for ( var currentKey in arguments ){
    pathArray.append( arguments[currentKey] );
}
```

I'm working on the PR now. (Please be patient as this will be my first one!)

**Comment by josephmchastain** [comment]
@elpete Do you have any branch naming conventions? Also, I noticed in the git history that the commit message for another compatibility issue for ACF is prepended with 'compat(ACF):' I'm guessing that's a convention that I should follow for my commit message as well? I think I have everything pretty much ready to go but want to make sure I don't make extra work for anyone by doing something incorrectly.

**Comment by elpete** [comment]
I squash and change commit messages as needed so you don’t need to worry about that. 👍🏻

**Comment by josephmchastain** [comment]
I'm trying to publish the branch with the fix and it says that I don't have write access to _coldbox-modules/cbPlaywright_. Anything I'm missing on my end?

**Comment by elpete** [comment]
Create a fork of the repo in your account and push to your fork. You will be able to make a pull request to here then.

**Comment by josephmchastain** [comment]
Ok, thanks Eric. I think the PR is submitted.

**Comment by elpete** [comment]
Fixed in v1.0.3
